tests/resources.py

```diff
@@ -52,8 +52,9 @@ class AstroidCacheSetupMixin:
     @classmethod
     def setup_class(cls):
         cls._builtins = AstroidManager().astroid_cache.get("builtins")
+        cls._transforms = AstroidManager.brain["_transform"]
```

Comment:
Why is the previous line instantiating an `AstroidManager`? Should that be done here as well?

Reply:
Because it's accessing the `astroid_cache`, not the `brain`. (I know...)

Reply:
Ah, never mind. I guess things are never simple with `astroid`...

Reply:
I guess I could be rewritten to use `AstroidManager()._transform`. Doesn't feel any clearer, but would be more consistent? 🤔

Reply:
I think this is fine!